main.go

```diff
@@ -223,6 +223,11 @@ func handlePost(res http.ResponseWriter, req *http.Request) {
 
 	date, _ := time.ParseInLocation("2006-01-02", dateStr, loc)
 
+	if date.Weekday() != 4 && !time.IsZero() {
```

Comment:
s/4/[time.Thursday](https://godoc.org/time#Weekday)/
s/time/date/ (that should have popped up when trying to build)